yearn/test/system/fixtures.js

```diff
@@ -21,3 +21,24 @@ module.exports.curveStrategyFixture = {
   // The fee the strategist will receive based on this Vault's performance.
   strategyPerformanceFee: 1000, // 10%
 }
+
+module.exports.saddleStrategyFixture = {
+  // Name of the vault for tBTCv2 Curve pool.
+  vaultName: "Curve tBTCv2 Pool yVault",
+  // Symbol of the vault for tBTCv2 Curve pool.
+  vaultSymbol: "yvCurve-tBTCv2",
+  // Total deposit limit of the vault for tBTCv2 Curve pool.
+  vaultDepositLimit: to1ePrecision(300, 15), // 0.3
+  // Amount of the deposit made by the depositor.
+  vaultDepositAmount: to1ePrecision(300, 15), // 0.3
+  // Amount of Synthetix staking rewards which should be allocated.
+  synthetixRewardsAllocation: to1e18(100000), // 100k
```

Comment:
Not used in Saddle strategy.

Reply:
Removed in https://github.com/keep-network/tbtc-v2/commit/045e79a6b12147241e18295132a5992350505be2.